config/300-function-manager.yaml

```diff
@@ -14,3 +14,12 @@ spec:
             image: github.com/vmware/dispatch/cmd/dispatch-server
             args:
             - function-manager
+            - --image-registry berndtj
```

Comment:
No thanks ;). This can be ${KO_DOCKER_REPO}

Reply:
Oh... yeah, I'm not sure I want to keep the config/*.yaml files.  Just one way to install.  But that's for another commit